CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+*.pt @rightscale/professional-services @rightscale/phoenix
+* @rightscale/professional-services
```

Comment:
I think the rule with * needs to be first as the last rule to match always wins.

Reply:
I would think that means professional-services gets notified on everything, and *.pt notifies both teams.

Reply:
It verifies each rule separately. It doesn't stop as soon as a rule matches, but rather at EOF. So the last match wins. So the current way you have it means the PS time will always be notified and the previous rules essentially don't matter.